src/battle_util.c

```diff
@@ -8347,7 +8345,7 @@ u8 IsMonDisobedient(void)
     if (BattlerHasAi(gBattlerAttacker))
         return 0;
 
-    if (IsBattlerModernFatefulEncounter(gBattlerAttacker)) // only false if illegal Mew or Deoxys
+    if (TRUE) // This used to be a check to see if the mon in question was a fateful encounter (Mew or Deoxys)
```

Comment:
I don't think this is necessary. It is just legacy code that we don't need to support.